apps/condo/domains/acquiring/schema/GeneratePaymentLinkService.test.js

```diff
@@ -0,0 +1,399 @@
+/**
+ * Generated by `createservice acquiring.GeneratePaymentLinkService`
+ */
+
+const conf = require('@condo/config')
+const faker = require('faker')
+const dayjs = require('dayjs')
+
+const {
+    generatePaymentLinkByTestClient,
+    updateTestAcquiringIntegrationContext,
+    makePayer,
+} = require('@condo/domains/acquiring/utils/testSchema')
+const {
+    updateTestBillingReceipt,
+} = require('@condo/domains/billing/utils/testSchema')
+const {
+    catchErrorFrom,
+} = require('@condo/domains/common/utils/testSchema')
+const {
+    PAYMENT_LINK_PATH,
+} = require('@condo/domains/acquiring/constants/links')
+
+const hostUrl = conf.SERVER_URL
+
+const callbacks = () => ({
+    successUrl: 'http://success.ru',
+    failureUrl: 'http://failure.ru',
+})
+
+const receiptData = () => ({
+    currencyCode: 'RUB',
+    amount: '100.23',
+    periodYear: 2022,
+    periodMonth: 1,
+    accountNumber: '7200123006',
+})
```

Comment:
best to move to test utils

Reply:
I would like to keep those small & test specific functions rather than move then in common test utils module